docs/docs/cmd/spo/site/site-recyclebinitem-list.md

```diff
@@ -0,0 +1,40 @@
+# spo site recyclebinitem list
+
+Lists items from recycle bin
+
+## Usage
+
+```sh
+m365 spo site recyclebinitem list [options]
+```
+
+## Options
+
+`-u, --siteUrl <siteUrl>`
+: URL of the site for which to retrieve the recycle bin items
+
+`--type [type]`
+: type of items which should be retrieved (listItems, folders, files)
```

Comment:
Let's start sentences with uppercase for consistency

Reply:
done